### PR TITLE
More Survival Tools Mod Series 4, Plus MShock24TilesetModded additions

### PR DESCRIPTION
**Pull request by chaosvolt**
Already? Yep.

Major Changes:
1. Highly belatedly took advantage of the tweaks to use actions, so now
the ember carrier doesn't need its own custom function. Lighting items
with it also takes a turn instead of being instant.

Item Tweaks:
1. Work gloves now have 20 encumbrence instead of 10.

Recipe Tweaks:
1. Recipe for hooded hat can use a bandana for the hood.
2. More tinder options.

### CODE REVIEW

**Comment by chaosvolt** [comment]
I had also pondered the idea of field-expedient tents using emergency blankets, modding in a tarp, etc. Most manuals imply you'd need either poles or nearby trees for even the most basic of tarp tent or poncho lean-to though.

That of course means that, for example, a tarp plus some spikes, 2 sticks, and enough string would have far more volume than a simple tent kit.

File: data/mods/More_Survival_Tools/items.json [hunk 1]
```diff
@@ -99,7 +99,20 @@
     "turns_per_charge": 600,
     "ammo": "tinder",
     "revert_to": "tinderbox",
-    "use_action": "TINDERBOX_LIT"
```
Comment (Rivet-the-Zombie) [inline data/mods/More_Survival_Tools/items.json]: If you're going to remove the reference to this function, you should remove the entire function from the C++ files in /src/.

**Comment by chaosvolt** [comment]
Meep. Very well then, I'll do so as well.

**Comment by chaosvolt** [comment]
And now, I hunger for content suggestions. owo

**Comment by stk2008** [comment]
I would love to see some kind of hunting hide.

I made a post on the forum regarding hunting and tracking.

If we had a hide we could place that allows us to enter that would be awesome.

I like the idea of tarp tents to that sounds really cool :)

**Comment by chaosvolt** [comment]
Ah, like a hunting blind? Would be interesting, but that might require code to allow terrain to conceal anyone inside it, unsure how the code would work.

Emergency blankets, folding ponchos, and rain coats could also work for tarp tents, but they'd need different setups. Especially since the easiest way to code them would be as constructions, and you'd want them to be removable. Putting stakes through a space blanket would likely be a bad idea if you want to re-use it, but there are ways to secure material without stakes. owo

**Comment by chaosvolt** [comment]
Also, one minor thing: If I add any terrain like tarp tents, this i gonna have to become a tileset update as well. I prefer not to leave transparency errors. ;w;

**Comment by Zireael07** [comment]
A suggestion: leave the tarp tents for another PRs. If they are too big/touch too many files, they tend to get merged much, much later.

**Comment by chaosvolt** [comment]
Ack. Possibly, though dedicating it to a separate PR will be annoying. Guess I can just add in the tarp for now and save the tents for a separate PR.

**Comment by chaosvolt** [comment]
I suspect that the changes to the source code will delay things even worse.

**Comment by chaosvolt** [comment]
What the...aw crap. What broke now? D:

**Comment by Coolthulhu** [comment]
Source code tweaks became retroactively fucked, yeah.

But it's an easy fix, because you only remove things.

**Comment by chaosvolt** [comment]
Ah. I was gonna say, I noticed 2 of my mod files have "no newline at end of file" issues. Should I fix that just in case? >w>

**Comment by chaosvolt** [comment]
Nope, guess it must be the retroactive fucking. ;w;

**Comment by Coolthulhu** [comment]
Unfuck it by merging the changes and fixing it then.

`iuse.h` changed and there's no way your changes will become mergeable if you don't address this.

**Comment by chaosvolt** [comment]
Ack, not sure how to do that. >.<

**Comment by Coolthulhu** [comment]
You have to switch to master branch, update it (pull from current master), switch back to your mod branch and merge master. Maybe pulling from current master to your branch would be enough.

**Comment by chaosvolt** [comment]
Ah, let me see...

**Comment by chaosvolt** [comment]
Error: Failed to check out branch. Wha...?

**Comment by chaosvolt** [comment]
Okay, this is starting to melt my brain. I should just make a new branch.

**Comment by chaosvolt** [comment]
Resolve merge conflict. Resolve merge conflict. God fucking damnit I would if I knew HOW.

**Comment by chaosvolt** [comment]
Fucking staring at a list of uncommited updates that it refuses to let me discard.

**Comment by Coolthulhu** [comment]
All this stuff is so much easier in console version of git... I could give you a complete list of commands if you had it.

**Comment by chaosvolt** [comment]
I've been fucking about in both Git Shell and the Github Windows thing. I THINK I might've forced it to update my master branch properly, dunno yet.

**Comment by Coolthulhu** [comment]
For some reason it looks like you forced your current master to be the one we had several versions ago.

I think at this point it would be easier if you just took updated your master branch properly (reset it to current HEAD if it is still wrong - if it has `point` and not `tripoint` in `iuse.h` it is wrong) and copied your mod files onto it. Then you could manually remove the tinder code.

**Comment by chaosvolt** [comment]
If I could figure out how to do any of that without breaking it... >_>

**Comment by Coolthulhu** [comment]
If you're afraid of breaking stuff, manually copy the entire folder first.
Git doesn't store repository state, except with files in the folder.

**Comment by chaosvolt** [comment]
And if I've ALREADY broken it? >_>

**Comment by Coolthulhu** [comment]
Then copy your mod files and redownload the repository.
And if it wrecked your mod files, you can try resetting back to the commit that still had them. Unless you did a rebase (you probably didn't, unless you had "rebase" in your command line/button), in which case they can be truly wrecked.

To reset, you need to find the hash of the last good commit and `git reset` to it. `gitk` will show you the log of current repo. You can then find your "Update 4" commit. Copy its SHA id and the `git reset [sha-id-here]` (without the `[` `]` ). This should un-commit everything up until that moment.
You can then view what you un-commited with `git diff`. Not sure what's the proper way to drop uncommited changes, so I just `git stash` and `git stash drop`.

**Comment by chaosvolt** [comment]
I'm already halfway through a new branch. ._.

**Pull request closed without merging**